graphql_compiler/schema_transformation/merge_schemas.py

```diff
@@ -333,3 +343,373 @@ def _process_generic_type_definition(generic_type, schema_id, existing_scalars,
         )
     merged_schema_ast.definitions.append(generic_type)
     type_name_to_schema_id[type_name] = schema_id
+
+
+def _add_cross_schema_edges(schema_ast, type_name_to_schema_id, scalars, cross_schema_edges,
+                            type_equivalence_hints, query_type):
+    """Add cross schema edges into the schema AST.
+
+    Each cross schema edge will be incorporated into the schema by adding vertex fields
+    with a @stitch directive to relevant vertex types. New fields will be named out_
```

Comment:
nit: consider quotes or backticks around `out_` and `in_`, and clarifying how the "new fields" relate to the cross-schema edges. Also consider hyphenating `cross-schema`.

How about something like:
The new fields corresponding to the added cross-schema edges will have names constructed from the edge name, prefixed with "out_" on the edge's outbound side, and "in_" on the edge's inbound side.